stubs/docker/docker/api/network.pyi

```diff
@@ -9,16 +9,16 @@ class NetworkApiMixin:
         self,
         name: str,
         driver: str | None = None,
-        options: dict[Incomplete, Incomplete] | None = None,
+        options: dict[str, Any] | None = None,
         ipam: IPAMConfig | None = None,
         check_duplicate: bool | None = None,
         internal: bool = False,
-        labels: dict[Incomplete, Incomplete] | None = None,
+        labels: dict[str, Any] | None = None,
```

Comment:
https://docs.docker.com/config/labels-custom-metadata/ explains that the keys are strings and the values are JSONable objects.